### PR TITLE
fixed some bad phrasing

### PR DESCRIPTION
**Pull request by ozkatz**
Tried to address most of your comments - currently only for the sizing guide, will fix the abuse commands in another PR

### CODE REVIEW

**Comment by ozkatz** [comment]
@arielshaqed addresses most comments in #1602

File: docs/architecture/sizing-guide.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ For high throughput, additional CPUs help scale requests across different cores.
 ### Network
 If using the data APIs such as the [S3 Gateway](overview.md#s3-gateway), 
 lakeFS will require enough network bandwidth to support the planned concurrent network upload/download operations.
+For most cloud providers, "stronger" machines (i.e. more expensive and usually with more CPU cores) also provide increased network bandwidth.
```
Comment (arielshaqed) [inline docs/architecture/sizing-guide.md:34]:
```suggestion
For most cloud providers, more powerful machines (i.e. more expensive and usually with more CPU cores) also provide increased network bandwidth.
```

**Review by arielshaqed — APPROVED** [review]
Thanks!

**Comment by arielshaqed** [comment]
Confirmed still _not_ to work using `make docs-serve`.  Back to the drawing board :triangular_ruler: :straight_ruler:

**Pull request merged**